docs/cli/kops_create.md

```diff
@@ -12,7 +12,8 @@ Create a resource:
   * secret  
   * federation  
 
-Create a cluster, instancegroup or secret using command line flags or YAML cluster spec. Clusters and instancegroups can be created using the YAML cluster spec.
+Create a cluster, instancegroup or secret using command line parameters or YAML config files.
+(Note: secrets cannot be created from YAML config files yet).
```

Comment:
I like this.

Maybe "or YAML configuration specification files" side-steps the issue?